packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/jni/JavaScriptObjectPropertyDescriptor.kt

```diff
@@ -0,0 +1,21 @@
+package expo.modules.kotlin.jni
+
+/**
+ * The property descriptor options for the property being defined or modified.
+ */
+enum class JavaScriptObjectPropertyDescriptor(val value: Int) {
```

Comment:
Maybe it makes sense to make it a nested enum of the `JavaScriptObject` class instead of such a long name.
```kt
class JavaScriptObject {
  enum class PropertyDescriptor { ... }
}
```
Then we can refer to it e.g.: `listOf(JavaScriptObject.PropertyDescriptor.Enumerable)`